hack/update-codegen.sh

```diff
@@ -32,27 +33,16 @@ ${REPO_ROOT_DIR}/hack/update-checksums.sh
 
 group "Kubernetes Codegen"
 
-# generate the code with:
-# --output-base    because this script should also be able to run inside the vendor dir of
-#                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
-#                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-  knative.dev/eventing/pkg/client knative.dev/eventing/pkg/apis \
-  "sinks:v1alpha1 eventing:v1alpha1 eventing:v1beta1 eventing:v1beta2 eventing:v1beta3 eventing:v1 messaging:v1 flows:v1 sources:v1beta2 sources:v1" \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
-
-# Deep copy config
-${GOPATH}/bin/deepcopy-gen \
-  -O zz_generated.deepcopy \
-  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt \
-  -i knative.dev/eventing/pkg/apis/config \
-  -i knative.dev/eventing/pkg/apis/messaging/config \
+kube::codegen::gen_helpers \
+  --boilerplate "${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt" \
```

Comment:
```suggestion
  --boilerplate "$(boilerplate)" \
```

You could use our function to get a boilerplate, with the current year. Changes to the year alone will be automatically rolled back.

Reply:
I'd make minimal changes for now, we can follow up?

Reply:
Sure. This is just a hint.